FalmouthFalcons/iteration-3
# Ticket4/jq

Description:
-Add description about what this pull request covers.

Number of Fixes:
Ticket 4

Related Ticket(s):
Ticket 4

Problem to Solve:
Showing all categories and total products per category

Proposed Changes:
Showing all categories and total products per category

Expected Behavior:
Showing all categories and total products per category

Steps to Test Solution
1) git fetch
2) git checkout ticket4/jq
3) rails db:migrate
rails db:seed

Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
